aiosmtpd/handlers.py

```diff
@@ -155,6 +155,29 @@ def handle_message(self, message):
 
 
 @public
+class AsyncMessage(Message):
+
```

Comment:
I'd like to see a docstring here, to briefly describe the uses of this class.  E.g. Why use this over the base class?